### PR TITLE
V2.6.4

### PR DESCRIPTION
**Pull request by kcw-grunt**

### CODE REVIEW

File: loafwallet/src/ModalPresenter.swift [hunk 1]
```diff
@@ -382,8 +382,24 @@ class ModalPresenter : Subscriber, Trackable {
...
         let settingsNav = UINavigationController()
-        let sections = ["Wallet", "Manage", "Support", "Blockchain"]
-        var rows = [
+        let sections = ["About", "Wallet", "Manage", "Support", "Blockchain"]
+        let rows = [
+            "About": [Setting(title: S.Settings.litewalletVersion, accessoryText: { [weak self] in
+                        return AppVersion.string
+                        }, callback: {}),
+                      Setting(title: S.Settings.litewalletEnvironment, accessoryText: { [weak self] in
+                        var envName = ""
```
Comment (mosadialiou) [inline loafwallet/src/ModalPresenter.swift]: Could set envName to EnvironmentVariables.EnvironmentName.release.rawValue and avoid the else block

File: loafwallet/src/ViewControllers/LoginViewController.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 import LocalAuthentication
-
+import FirebaseCrashlytics
```
Comment (mosadialiou) [inline loafwallet/src/ViewControllers/LoginViewController.swift]: unused import. should be removed

File: loafwallet/src/WalletManager+Auth.swift [hunk 3]
```diff
@@ -451,6 +452,34 @@ extension WalletManager : WalletAuthenticator {
         }
     }
     
+   func deleteWalletDatabase(pin: String = "forceWipe") -> Bool {
```
Comment (mosadialiou) [inline loafwallet/src/WalletManager+Auth.swift]: `forceWipe` should be an **Enum** or at least a **Constant**

File: loafwallet/AppDelegate.swift [hunk 4]
```diff
@@ -96,5 +98,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, shouldRestoreApplicationState coder: NSCoder) -> Bool {
         return true
     }
+    
+    private func setFirebaseConfiguration() {
+        
+        if let path = Bundle.main.path(forResource: "GoogleService-Info", ofType: "plist") {
```
Comment (mosadialiou) [inline loafwallet/AppDelegate.swift]: Is this whole check necessary?

File: loafwallet/AppDelegate.swift [hunk 5]
```diff
@@ -96,5 +98,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+                var dict = NSDictionary(contentsOfFile: path)
+
+                #if Debug || Testflight
+                    if let debugPath = Bundle.main.path(forResource: "GoogleService-iOS-Beta-Info", ofType: "plist") {
+                        dict = NSDictionary(contentsOfFile: debugPath)
+                        if dict?.write(toFile: path, atomically: true) == nil {
```
Comment (mosadialiou) [inline loafwallet/AppDelegate.swift]: What are you trying to do here?

File: loafwallet/GoogleService-Info.plist [hunk 6]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mosadialiou) [inline loafwallet/GoogleService-Info.plist]: This file has been explicitly gitignored so it should not be present in this PR.

File: loafwallet/GoogleService-iOS-Beta-Info.plist [hunk 7]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mosadialiou) [inline loafwallet/GoogleService-iOS-Beta-Info.plist]: This file has been explicitly gitignored so it must not be present in this PR.

File: loafwallet/TabBarViewController.swift [hunk 8]
```diff
@@ -87,7 +87,7 @@ class TabBarViewController: UIViewController, Subscriber, Trackable, UITabBarDel
             viewControllers.append(controller)
         }
         
-        self.updateTimer = Timer.scheduledTimer(withTimeInterval: 120.0, repeats: true) { timer in
+        self.updateTimer = Timer.scheduledTimer(withTimeInterval: 30.0, repeats: true) { timer in
```
[the remark this answers is not in the document]
Comment (kcw-grunt) [reply loafwallet/TabBarViewController.swift]: the refresh was invisible before (the conversion rate would update) with the new price label it needs to be more fresh

File: loafwallet/src/ModalPresenter.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/src/ModalPresenter.swift]: this was just for triggering the CI. Closing PR

File: loafwallet/src/ViewControllers/LoginViewController.swift (hunk 2 above)
Comment (kcw-grunt) [reply loafwallet/src/ViewControllers/LoginViewController.swift]: this was just for triggering the CI. Closing PR

File: loafwallet/src/WalletManager+Auth.swift (hunk 3 above)
Comment (kcw-grunt) [reply loafwallet/src/WalletManager+Auth.swift]: this was just for triggering the CI. Closing PR

File: loafwallet/AppDelegate.swift (hunk 4 above)
Comment (kcw-grunt) [reply loafwallet/AppDelegate.swift]: this was just for triggering the CI. Closing PR

File: loafwallet/AppDelegate.swift (hunk 5 above)
Comment (kcw-grunt) [reply loafwallet/AppDelegate.swift]: this was just for triggering the CI. Closing PR

File: loafwallet/GoogleService-Info.plist (hunk 6 above)
Comment (kcw-grunt) [reply loafwallet/GoogleService-Info.plist]: this was just for triggering the CI. Closing PR

File: loafwallet/GoogleService-iOS-Beta-Info.plist (hunk 7 above)
Comment (kcw-grunt) [reply loafwallet/GoogleService-iOS-Beta-Info.plist]: this was just for triggering the CI. Closing PR

**Comment by kcw-grunt** [comment]
@mosadialiou lets discuss on the issues explicitly. Alot of the cruft in android is present here as well. This PR was just a test to trigger the CI

**Pull request closed without merging**